### PR TITLE
Update README with mint instructions

### PR DESCRIPTION
**Pull request by levibostian**
Update the README to include instructions for both SPM plugin and mint installation.

Open to ideas! Please offer suggestions if you have them.

### CODE REVIEW

**Comment by levibostian** [comment]
Note: Good idea to not merge this PR until [this one is merged](https://github.com/csjones/lefthook-plugin/pull/4).

**Review by csjones — APPROVED** [review]
Looks good! Thank you for the contributions 😃

**Pull request merged**
